libs/libglog/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=glog
+PKG_VERSION:=0.3.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
This will eventually clash with other packages, best is to use `PKG_SOURCE_PROTO:=git`

Reply:
Done